uprotocol/uattributes.proto

```diff
@@ -46,38 +46,43 @@ message UAttributes {
     // Message type (publish, request, response)
     UMessageType type = 2;
 
+    // The source (address) of the message. For published events this is the producers publish topic
+    // for requests it is the calling uE respose topic (who sent the request). For responses this is the
+    // method URI from the uService. 
+    UUri source = 3;
+
     // Destination UUri for a message used for unicast message types 
     // (notification, request, response)
-    optional UUri sink = 3;
+    optional UUri sink = 4;
```

Comment:
My understanding is that in proto3 message typed properties are always optional.

Is there a particular reason why some of them are explicitly marked `optional` while others are not?

Reply:
Nope, removed you are correct.